bin/smash

```diff
@@ -32,18 +32,19 @@ def main():
    import datetime
    import smashbox.script
    import smashbox.no_engine
-
+   from smashbox.utilities import  oc_webdav_url,check_url, time_now, get_data_from_json, rm_file_dir, finish_test
    parser=smashbox.script.arg_parser(description='Run the tests and smash the box')
 
    parser.add_argument('tests', metavar='test', type=str, nargs='*',
                       help='single test (file) or test collection (directory)')
 
    parser.add_argument('--keep-state', dest="keep_state",action="store_true", help='incremental test on top of existing files: keep current state of the account both locally and on the server')
-   parser.add_argument('--keep-going',dest="keep_going",action="store_true",default=False,help='do not abort the execution on the first failed testcase')
+   parser.add_argument('--stop-going',dest="stop_going",action="store_false",default=True,help='abort the execution on the first failed testcase')
```

Comment:
A little insight on this rather critical change would be nice.
Why would you not want to break apart by default when there was an error?